lib/wda/utils.js

```diff
@@ -198,6 +198,7 @@ CODE_SIGN_IDENTITY = ${signingId}
  * @property {string} isRealDevice - Equals to true if the current device is a real device
  * @property {string} udid - The device UDID.
  * @property {string} platformVersion - The platform version of OS.
+ * @property {string} platformNameLowerCase - The platform name of iOS, tvOS
```

Comment:
I'd rather keep the name as `platformName`. You could always change the value to lower case if you need